python/kfserving/README.md

```diff
@@ -30,4 +30,68 @@ Requirement already satisfied: argparse>=1.4.0 in /Library/Frameworks/Python.fra
 Installing collected packages: kfserver
   Running setup.py develop for kfserver
 Successfully installed kfserver
-```
\ No newline at end of file
+```
+
+## Development
+
+To install development requirements
+
+```bash
+pip install -e .[test]
```

Comment:
Just curious, what does the test part do?

Reply:
This is a trick to allow you to install the test requirements specified in the setup.py. Seems to be a convention - by pointing to one part of the extra_requirements Dict.